launcher/src/main/java/io/snappydata/tools/QuickLauncher.java

```diff
@@ -124,6 +124,15 @@ private int start(final String[] args) throws IOException, InterruptedException
     // add java path to command-line first
     commandLine.add(System.getProperty("java.home") + "/bin/java");
     commandLine.add("-server");
+    // https://github.com/airlift/jvmkill is added to libgemfirexd.so
+    // adding agentpath helps kill jvm in case of OOM. kill -9 is not
+    // used as it fails in certain cases
+    String arch = System.getProperty("os.arch");
+    if(arch.contains("64") || arch.contains("s390x")){
```

Comment:
@hemanthmeka Ok, since this is launcher which does not have access to SharedLibrary or Native classes, its better to directly use JNA's com.sun.jna.Platform.is64Bit() rather than os.arch check.